examples/sample-app/README.md

```diff
@@ -82,8 +97,12 @@ the present working directory is the same directory as this README.
         $ ./pullimages.sh
 
 2. Launch an all-in-one `openshift` instance
-
+                                                                                               
         $ sudo openshift start &> logs/openshift.log &
```

Comment:
Can we change this line as well, I usually c&p this and there's no logs dir in place. I'd propose removing logs from here :smile: